### DISCUSSION TITLE
[Announcement] Final exam is graded

### DISCUSSION THREAD

**Discussion by yeonhee-ryou** [prosyslab-classroom/cs524-program-analysis#248] [Announcements]
Hi all,
The final exam is graded.

You can check your score and criteria for each problem at Gradescope.
If you want to make any claims for your score, send a "Regrade request(재채점 요청)" in Gradescope system, or an email cs524.ta@prosys.kr until this **Friday(12.15), 23:59**.

FYI, here are the statistics for this exam.
Average: 34
Standard Deviation: 23

**Comment by gayeun4195** [comment]
Could I get the distribution (like a histogram) for the exam?
